lib/generators/rodauth/templates/app/views/rodauth/tailwind/change_login.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :title, rodauth.change_login_page_title %>
+<div class="dark:bg-gray-900 flex items-center justify-center h-screen">
+  <div class="max-w-sm">
+    <h3 class="mb-3 text-3xl font-semibold text-center dark:text-gray-200"> Change Your Login Email Addres </h3>
```

Comment:
With the goal of avoid hardcoded english text, we could use `rodauth.change_login_page_title` here, which says "Change Login" by default. And similar for other forms with titles.

Reply:
fixed with: [5bb294c286b104c5c2d6f104e5d2a3f2188f4dc0](https://github.com/janko/rodauth-rails/pull/114/commits/5bb294c286b104c5c2d6f104e5d2a3f2188f4dc0)